docs/reference/analysis/charfilters/mapping-charfilter.asciidoc

```diff
@@ -4,6 +4,16 @@
 A char filter of type `mapping` replacing characters of an analyzed text
 with given mapping.
 
+[horizontal]
+`mappings`::
+
+    A list of mappings to use.
+
+`mappings_path`::
+
+    A path (either relative to `config` location, or absolute) to a mappings
+    file configuration.
```

Comment:
Maybe add a note about how the relative locations are preferred because the security manager refuses to read from locations that Elasticsearch hasn't granted itself permissions to read from?